crates/nu-cmd-extra/src/extra/formats/to/html.rs

```diff
@@ -175,8 +182,17 @@ fn get_theme_from_asset_file(
     let th = asset
         .themes
         .into_iter()
-        .find(|n| n.name.eq_ignore_case(theme_name)) // case insensitive search
-        .unwrap_or_default();
+        .find(|n| n.name.eq_ignore_case(theme_name)); // case insensitive search
+
+    let th = match th {
+        Some(t) => t,
+        None => {
+            return Err(ShellError::TypeMismatch {
```

Comment:
Minor nit, those errors should probably use something other than the problematic `TypeMismatch`